src/nft/utils/ipfs.ts

```diff
@@ -0,0 +1,31 @@
+export function IsValidIpfsUrl(url: string): boolean {
```

Comment:
Should we put these two functions in their own files? 

`/utils/IsValidIpfsUrl/.ts`

`/utils/convertIpfsToHttps.ts`

Reply:
i can, they were both so related i figured a single ipfs file would work well

Reply:
i think it is fine in one file